flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/CliClientTest.java

```diff
@@ -144,18 +138,13 @@ public void testUseNonExistingCatalog() throws Exception {
 			.build();
 
 		InputStream inputStream = new ByteArrayInputStream("use catalog cat;\n".getBytes());
-		CliClient cliClient = null;
 		SessionContext sessionContext = new SessionContext("test-session", new Environment());
 		String sessionId = executor.openSession(sessionContext);
 
-		try (Terminal terminal = new DumbTerminal(inputStream, new MockOutputStream())) {
-			cliClient = new CliClient(terminal, sessionId, executor, File.createTempFile("history", "tmp").toPath());
-			cliClient.open();
+		try (Terminal terminal = new DumbTerminal(inputStream, new MockOutputStream());
+			 CliClient client = new CliClient(terminal, sessionId, executor, File.createTempFile("history", "tmp").toPath())) {
```

Comment:
nit: fix indention here and below, seems like a typical copy paste mistake. how about we create a method to reduce duplicate code (e.g. for the tmp file)?

Reply:
> nit: fix indention here and below, seems like a typical copy paste mistake. how about we create a method to reduce duplicate code (e.g. for the tmp file)?

Maybe Idea's indention is using space. Any config could change to tab?